src/Carter/Carter.csproj

```diff
@@ -13,21 +13,27 @@
         <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
         <MinVerSkip Condition="'$(Configuration)' == 'Debug'">true</MinVerSkip>
         <PackageReadmeFile>README.md</PackageReadmeFile>
+        <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+        <NoWarn>RS2008</NoWarn>
     </PropertyGroup>
     <ItemGroup>
         <None Include="..\..\media\carterlogo.png" Pack="true" PackagePath="\" />
         <None Include="..\..\README.md" Pack="true" PackagePath="\"/>
+        <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
Excuse my ignorance again - what does this do?

Reply:
Please don't hold back with questions – it's important you understand the code I added :)

To answer your question: It's not enough to add the code to the project. To make analyzers work inside a NuGet package, they must be placed in a specific directory inside the package, in this case the `analyzers/dotnet/cs` directory, since it is a C# analyzer.